tutor/src/routes.coffee

```diff
@@ -1,8 +1,7 @@
 async = require './helpers/webpack-async-loader'
 
 getCourseListing = ->
-  {CourseListing} = require './components/course-listing'
-  CourseListing
+  require './components/course-listing'
```

Comment:
and yah, this doesn't belong in here.  it confused me also :(  I'm going to rebase it out and make the change on #1400